config/etc/cloud/cloud.cfg.d/10_snappy.cfg

```diff
@@ -23,6 +26,3 @@ grub_dpkg:
 
 # apt_pipelining logs warning on failed write to /etc/apt/apt.conf.d
 apt_pipelining: "none"
-
-# 'apt_configure' module should not run as apt is not used.
-apt_configure_enabled: False
```

Comment:
hmm, also ... is there a git add missing for the mentioned ds-identify.cfg file ?